ui/js/page/wallet.js

```diff
@@ -16,20 +16,20 @@ var AddressSection = React.createClass({
       this.setState({
         address: address,
       });
-    });
+    }.bind(this));
```

Comment:
I did not realized that arrow functions work differently from normal functions. I assumed that was just the new way of writing them.